src/docs/processing-tickets.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Processing Issues & PRs"
+---
+
+This document describes Sentry's process for handling issues and PRs—let's use "tickets" to refer to both together, if that's okay—that are created by external contributors in our public GitHub repos.
+
+## 1. Validate
+
+*SLA: 1 hour*
+
+The Open Source Team maintains [a bot](https://github.com/getsentry/.github/blob/main/.github/workflows/validate-new-issue.yml) that validates incoming tickets. Invalid tickets are closed; valid tickets are given the label `Status: Unrouted`. The bot is installed on three repos: `sentry`, `sentry-docs`, and `onpremise`.
+
+## 2. Route
+
+*SLA: 72 hours*
+
+Each ticket is routed to a single Engineering team based on our internal [Product and Service Registry](https://www.notion.so/Product-and-Service-Registry-f90c702816794dcaa7f81ef8e79e40ea):
+
+1. Most repos are owned by a single Engineering team, so tickets opened in those repos are already routed.
+2. For the `sentry` repo, the Open Source Team is responsible for routing to other teams.
+3. For the `sentry-docs` repos, the Documentation team is responsible for routing to other teams as necessary.
+
+Routing in the `sentry` and `sentry-docs` repos is done with `Team: Foo` labels and a comment mentioning `@getsentry/team`, with the "Team" field in the Registry being the source of truth for who owns what.
+
+Some of our code is covered by a [`CODEOWNERS`](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) file; most is not. For the code that is in `CODEOWNERS`, GitHub will auto-assign a GitHub Team to review the PR. All tickets should get a `Team` label and a mention, regardless of whether they are also assigned directly. If and when GitHub allows for [assigning issues to teams](https://github.com/dear-github/dear-github/issues/170), we can revisit the use of labels/mentions vs. direct assignments.
+
+When a ticket is routed, remove `Status: Unrouted` and add `Status: Untriaged`.
```

Comment:
I think these two should be two separate `<Warning>` or `<Note>` sections to bring attention to them.

Reply:
I think one Alert is enough?

---------------

<img width="701" alt="Screen Shot 2021-03-05 at 8 37 23 AM" src="https://user-images.githubusercontent.com/134455/110122774-06551780-7d8e-11eb-92c4-a2cbf89b52d1.png">

Reply:
Done in e5c2a55e1df25f03f9b1c34a9f256812ddba0427.